signbank/dictionary/adminviews.py

```diff
@@ -713,6 +713,7 @@ def get_video_details(request):
     writer = csv.writer(response)
 
     csv_queryset =GlossVideo.objects.filter(is_public=True)
+    # return render_to_csv_response(csv_queryset)
```

Comment:
Are you deliberately leaving this comment?

Reply:
No, I'm not. Must have slipped out when I was removing other comments. Will remove it :smile: